content/en/blog/release-v1.2/index.md

```diff
@@ -0,0 +1,108 @@
+#  focal_point = "Center"
++++
+
+
+
+**KubeEdge** is an open source system extending native containerized application orchestration and device management to hosts at the Edge. It is built upon Kubernetes and provides core infrastructure support for networking, application deployment and metadata synchronization between cloud and edge. It also supports MQTT and allows developers to author custom logic and enable resource constrained device communication at the Edge.  
+
+## **Today we announce the v1.2 release of KubeEdge.**
+
+
+On February 9th, the KubeEdge community is proud to announce the availability of KubeEdge 1.2. This release includes a major upgrade on reliability, which includes more reliable message delivery from cloud to edge, component Config API, edge nodes auto-registration, Kubernetes v1.17.1 support, and 30+ fixes.
+
+Please refer to https://github.com/kubeedge/kubeedge/blob/release-1.2/CHANGELOG-1.2.md for a full list of features in this release, and the following for some highlights.
+
+{{% alert note %}}
+Check out the release here:  [Release v1.2](https://github.com/kubeedge/kubeedge/releases/tag/v1.2.0)  
+{{% /alert %}}
+
+{{% alert note %}}
+Instructions on how to setup KubeEdge can be found [here](https://github.com/kubeedge/kubeedge#usage)  
+{{% /alert %}}
+
+### A major upgrade on Cloud-Edge transmission reliability
+
+In an Edge Computing scenario, the instability of edge network could cause the reliability issues of edge’s communication to cloud, which could further cause data loss during communication. To improve Cloud-Edge transmission reliability, KubeEdge v1.2 includes following update：
+1. Added a verification mechanism for application layer message sending. There is a handshake mechanism built in to acknowledge (via ACK message) the successful state synchronization between cloud and edge. If the acknowledgement fails due to some reason, an application layer code will trigger the retransmission mechanism to re-synchronize the state.
+2. Implemented persistent cloud side collaborative messages. During the cloud-edge status synchronization process, cloud will record in real time the latest message version number (ResourceVersion) of each edge node that is successfully synchronized, and persist it to Kubernetes in the form of CRD. This mechanism can ensure the order and the continuity of message after a cloud failure or an edge node offline restart, avoiding cloud/edge inconsistent status caused by resending old messages.
+3. Implemented a periodic check for cloud-edge data to ensure consistency. Based on the above two features, KubeEdge 1.2 includes a new module that synchronizes Controller to CloudCore. This module periodically checks the synchronization status of edge nodes, compares the information of resources in Kubernetes, and synchronizes states, and ensure the ultimate state consistency between cloud and edge. 
```

Comment:
md format should be used here, otherwise line breaks will be lost in website. Preview page: https://deploy-preview-60--privatekubeedge.netlify.com/en/blog/release-v1.2/
- a
- b